tests/__snapshots__/index.spec.js.snap

```diff
@@ -161,7 +161,7 @@ Object {
     "test": "avec 50% paramètre",
   },
   "number.withUSDFormat": Object {
-    "test": "avec US$ 1,234.00 paramètre",
+    "test": "avec $1,234.00 paramètre",
```

Comment:
nope, run test with node 10

Reply:
ok tests passes. I don't understand anymore.

`$` is node 10 ?
`US$` is node 9 ?

Reply:
I don't know :/ On local , with node 10, I've the same issue that previous, ...

Reply:
```
➜  k-intl git:(upVersion) nvm use 10
Now using node v10.4.0 (npm v6.1.0)
➜  k-intl git:(upVersion) yarn test
yarn run v1.7.0
$ jest
 PASS  tests/index.spec.js
 PASS  src/utils/utils.spec.js

Test Suites: 2 passed, 2 total
Tests:       5 passed, 5 total
Snapshots:   6 passed, 6 total
Time:        1.05s
Ran all test suites.
✨  Done in 1.47s.
```

Reply:
```
➜  k-intl git:(upVersion) nvm use 9
Now using node v9.8.0 (npm v5.6.0)
➜  k-intl git:(upVersion) yarn test
yarn run v1.7.0
$ jest
 FAIL  tests/index.spec.js
  ● src/format › should format message with FR Locale

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

    @@ -16,11 +16,11 @@
        },
        "number.withPercentParam": Object {
          "test": "avec 50% paramètre",
        },
        "number.withUSDFormat": Object {
    -     "test": "avec $1,234.00 paramètre",
    +     "test": "avec US$ 1,234.00 paramètre",
        },
        "number.withoutParam": Object {
          "test": "aucun paramètre",
        },
        "plural.simple.noparam": Object {

      49 | const tester = (lang, locale) =>
      50 |   it(`should format message with ${lang} Locale`, () => {
    > 51 |     expect(keyValueResources(lang, format(lang, locale, customFormats))).toMatchSnapshot()
      52 |     expect(keyValueResources(lang, format(lang, undefined))).toMatchSnapshot()
      53 |   })
      54 |

      at Object.<anonymous> (tests/index.spec.js:51:74)

 › 1 snapshot test failed.
 PASS  src/utils/utils.spec.js

Snapshot Summary
 › 1 snapshot test failed in 1 test suite. Inspect your code changes or run `yarn test -u` to update them.

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 4 passed, 5 total
Snapshots:   1 failed, 5 passed, 6 total
Time:        1.237s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  k-intl git:(upVersion)
```